src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -674,14 +923,14 @@ StaticLLMPipeline::StaticLLMPipeline(
     }
 };
 
-StaticLLMPipeline::StaticLLMPipeline(
+StatelessLLMPipeline::StatelessLLMPipeline(
```

Comment:
Oh jez, to invasive, let's keep it `StaticLLMPipeline`, what's the problem with this?